src/hexo/themes/documentation/scripts/external-link-filter.js

```diff
@@ -0,0 +1,18 @@
+/* globals hexo */
+const cheerio = require('cheerio');
+const updateRelAttribute = (data) => {
+    const $ = cheerio.load(data.content, { decodeEntities: false });
+
+    $('a').each(function () {
+        if ($(this).attr('href') && $(this).attr('rel') === 'external') { //eslint-disable-line no-invalid-this
```

Comment:
Disable the eslint rule at the file level. No point in having it in two lines in a row for the amount of code we have here.